src/pal/src/eventprovider/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 2.8.12.2)
+
+if(CMAKE_SYSTEM_NAME STREQUAL Linux)
+project(eventprovider)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)   
+  
+  add_definitions(-DPAL_STDCPP_COMPAT=1)
+  include_directories(${COREPAL_SOURCE_DIR}/inc/rt)
+  add_compile_options(-fPIC)
+
+
+   add_library(eventprovider1
+"lttng/eventprovdotnetruntime.cpp"
+)
+   add_library(eventprovider2
+"lttng/eventprovdotnetruntimeprivate.cpp"
+)
+   add_library(eventprovider3
+"lttng/eventprovdotnetruntimerundown.cpp"
+)
+   add_library(eventprovider
+      STATIC
+"lttng/eventprovdotnetruntimestress.cpp"
+)
+
+target_link_libraries(eventprovider
+     eventprovider1
+     eventprovider2
+     eventprovider3
+)
+
+
+# Install the static PAL library for VS
+install (TARGETS eventprovider DESTINATION lib)
```

Comment:
The coding style of this file does not seem to have space before open parenthesis.